### PR TITLE
Task #4 - Implement eco projects page

### PR DESCRIPTION
**Pull request by ArturSydor**

### CODE REVIEW

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
please add group into readme file

**Comment by ArturSydor** [comment]
> please add group into readme file

added

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
The task #3 is tests. Please create tests and submit new PR

**Comment by ArturSydor** [comment]
> create

Conflicts were resolved and this PR is also ready now

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+**Student:** Artur Sydor
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:1]: add correct name for pull request to understand what task it is for.

Comment (ArturSydor) [reply README.md:1]: PR name updated

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**